SS14.Admin/Pages/Tables/BansTable.cshtml

```diff
@@ -115,7 +118,21 @@
     </tbody>
 </table>
 
+<script>
+    function showConfirmation(banId) {
+        var confirmationButton = document.getElementById('confirmation-button-' + banId);
+        var unbanButton = document.getElementById('unban-form-' + banId).querySelector('button[type="button"]');
```

Comment:
Don't use var. You should use let or in this case, since the variables don't get reassigned you can use const.
https://youtube.com/shorts/ZRjmGq1gAEQ?feature=shared

Reply:
donzo